### DISCUSSION TITLE
Навыки

### DISCUSSION THREAD

**Discussion by registant** [mindryder/ATLAS-RU#6] [НАЙДЕННЫЕ ОШИБКИ ПЕРЕВОДА]
![Screenshot_1358](https://user-images.githubusercontent.com/123338303/214794040-cf470d35-3bda-44f1-9a73-2ad8524cb9bd.jpg)
![Screenshot_1359](https://user-images.githubusercontent.com/123338303/214794043-b850b0f5-f0f7-4547-b547-e997d9e6aa24.jpg)

**Comment by mindryder** [comment]
Исправлено
